nvtabular/workflow/node.py

```diff
@@ -252,22 +342,44 @@ def parents_schema(self):
 
     @property
     def input_columns(self):
+        if not self.input_schema:
+            raise RuntimeError(
+                "The input columns aren't computed until the workflow "
+                "is fit to a dataset or input schema."
+            )
+
         if self.selector:
+            # To maintain column groupings
             return self.selector
```

Comment:
Do we still need to return the selector here? What's using this that needs the groupings? Could it ask for the selector directly?

Reply:
`Workflow` runs operator transforms with the result of the `input_columns` method as the selector, and the ops need the grouping, so we do in fact have to return the selector here when it's present.